substrate/frame/core-fellowship/src/tests/integration.rs

```diff
@@ -19,28 +19,19 @@
 
 #![allow(deprecated)]
 
-use frame_support::{
-	assert_noop, assert_ok, derive_impl, hypothetically, hypothetically_ok, ord_parameter_types,
-	pallet_prelude::Weight,
-	parameter_types,
-	traits::{ConstU16, EitherOf, IsInVec, MapSuccess, NoOpPoll, TryMapSuccess},
+use frame::{
+	testing_prelude::{storage::root as storage_root, TestExternalities, *},
```

Comment:
```suggestion
	testing_prelude::{storage::root as storage_root, *},
```

`TestExternalities` is not needed here since you're already importing everything inside the prelude